mypy_django_plugin/transformers/models.py

```diff
@@ -360,7 +358,7 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
     def try_generate_related_manager(
         self, related_model_cls: Type[Model], related_model_info: TypeInfo
     ) -> Optional[Instance]:
-        manager = related_model_cls._meta.managers_map["objects"]
+        manager = related_model_cls._meta.managers_map["_default_manager"]
```

Comment:
Changed from `objects` to `_default_manager` with https://docs.djangoproject.com/en/dev/topics/db/queries/#using-a-custom-reverse-manager as justification ("By default the RelatedManager used for reverse relations is a subclass of the default manager for that model.")